WordPress/Tests/WhiteSpace/PrecisionAlignmentUnitTest.css

```diff
@@ -0,0 +1,5 @@
+#login-container {
+    margin-left: -225px;
```

Comment:
Yes it is, as that will correctly be replaced by a tab by the `Generic.WhiteSpace.DisallowSpaceIndent`, so is not the concern of this sniff.